### PR TITLE
Added argos.co.uk

### PR DESCRIPTION
**Pull request by crleonard**
### Checklist
- [x] The PR isn't documenting something that would be a common practice among password managers (e.g. minimal length of 6)
- [x] The top-level JSON objects are sorted alphabetically
- [x] There are no [open pull requests](https://github.com/apple/password-manager-resources/pulls) for the same update.

#### for websites-with-shared-credential-backends.json
- [ ] There's evidence the domains are related (SSL certificates, DNS entries, valid links between sites, legal documents etc.)
- [ ] The new group serves login pages on each of the included domains, and those login page accept accounts from the others. (For example, we don't associate `google.co.il` to `google.com`, because `google.co.il` redirects to `accounts.google.com` for authentication.)

#### for change-password-URLs.json
- [ ] There is no Well-Known URL for Changing Passwords (`https://example.com/.well-known/change-password`)
- [ ] The URL either makes the experience better or no worse than being directed to just the domain in a non-logged-in state

#### for password-rules.json
- [x] The given rule isn't particularly standard and obvious for password managers
- [x] Generated passwords have been tested from this rule using the [Password Rules Validation Tool](https://developer.apple.com/password-rules/)
- [x] Information has been included about the website's requirements (eg. screenshots, error messages, steps during experimentation, etc.)

### CODE REVIEW

**Comment by rmondello** [comment]
@crleonard How did you determine these were the rules? Are there rules stated on the website?

**Comment by crleonard** [comment]
Hey, I went through the sign up process and the rules are displayed at the beginning on a page.

File: quirks/password-rules.json [hunk 1]
```diff
@@ -20,6 +20,9 @@
     "angieslist.com": {
         "password-rules": "minlength: 6; maxlength: 15;"
     },
+    "argosco.uk": {
+        "password-rules": "minlength: 6; required: lower, upper, digit;"
```
Comment (jumoru) [inline quirks/password-rules.json:24]: Is there a typo in this URL? [argosco.uk](argosco.uk) gives me an error message (404). Shouldn't this be [argos.co.uk](argos.co.uk) as stated in the title of pull request?

Comment (crleonard) [reply quirks/password-rules.json:24]: Just pushed new commit.

**Comment by rmondello** [comment]
@crleonard Hi! Sorry for the delay! Trying to get this merged. :)

Can you fix up the merge conflict?

Can you also verify that the rule is right? It says that the website requires at least one letter or a digit, as opposed to requiring one of a digit, one of a lower, and one of a number. That is, your rule says these are acceptable passwords:

aaaaaa
111111

**Comment by rmondello** [comment]
Closing due to inactivity. Feel free to re-open any time!

**Pull request closed without merging**